apps/web/cookie.config.ts

```diff
@@ -101,27 +110,31 @@ export default {
           Status: 'The consent cookies stores the user’s state of consent for cookies on our page.',
           PrivacyPolicy: '/PrivacyPolicy',
           Lifespan: '4 days',
+          accepted: false,
```

Comment:
Is it necessary to set it to false? I feel like it should only be set if we want it to be active